<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VikasAssistantClass</class>
 <widget class="QMainWindow" name="VikasAssistantClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VikasAssistant</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lbCostumeName">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="font">
         <font>
          <family>Vivaldi</family>
          <pointsize>14</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerItem</enum>
        </property>
        <property name="movement">
         <enum>QListView::Static</enum>
        </property>
        <property name="uniformItemSizes">
         <bool>false</bool>
        </property>
        <property name="selectionRectVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="openFileAction"/>
    <addaction name="closeAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="openFileAction">
   <property name="text">
    <string>Открыть...</string>
   </property>
  </action>
  <action name="closeAction">
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="VikasAssistant.qrc"/>
 </resources>
 <connections/>
</ui>
